ivadomed/loader/loader.py

```diff
@@ -51,7 +51,7 @@ def load_dataset(data_list, bids_path, transforms_params, model_params, target_s
     slice_filter_params and object_detection_params see :doc:`configuration_file`.
     """
     # Compose transforms
-    tranform_lst, _ = imed_transforms.preprare_transforms(transforms_params, requires_undo)
+    tranform_lst, _ = imed_transforms.preprare_transforms(copy.deepcopy(transforms_params), requires_undo)
```

Comment:
@andreanne-lemay : I ve added this deepcopy otherwise, when we arrive [here](https://github.com/ivadomed/ivadomed/blob/270864fa3a1674e1ab4f2d2a57aa933a8ce8a49a/ivadomed/loader/loader.py#L103), the ROICrop (for instance) is no longer in this list because of the `del` that are performed within `prepare_transforms`.

Reply:
I may have a stupid question but ... why is the name `preprare_transforms` instead of `prepare_transforms` ?

Reply:
Not a stupid question --> Rather a "stupid charley" :-)

Reply:
Hahah wow :') this is actually my fault, I defined the function this way :'). I updated the code!